<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.23.0 by Michael Rose
  Copyright 2013-2020 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
<html lang="en" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>Papers &amp; Preprints - Xu lab@CBD_SCS_CMU</title>
<meta name="description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search results) and in your feed.xml site description.">



<meta property="og:type" content="website">
<meta property="og:locale" content="en_US">
<meta property="og:site_name" content="Xu lab@CBD_SCS_CMU">
<meta property="og:title" content="Papers &amp; Preprints">
<meta property="og:url" content="http://localhost:4000/papers/">


  <meta property="og:description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search results) and in your feed.xml site description.">












<link rel="canonical" href="http://localhost:4000/papers/">




<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": null,
      "url": "http://localhost:4000/"
    
  }
</script>







<!-- end _includes/seo.html -->



<!-- https://t.co/dKP3o1e -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5/css/all.min.css">

<!--[if IE]>
  <style>
    /* old IE unsupported flexbox fixes */
    .greedy-nav .site-title {
      padding-right: 3em;
    }
    .greedy-nav button {
      position: absolute;
      top: 0;
      right: 0;
      height: 100%;
    }
  </style>
<![endif]-->


    <!-- start custom head snippets -->

<!-- insert favicons. use https://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

  </head>

  <body class="layout--splash">
    <nav class="skip-links">
  <h2 class="screen-reader-text">Skip links</h2>
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    <div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <a class="site-title" href="/">Xu lab@CBD_SCS_CMU</a>
        <ul class="visible-links">
                
                
                    <li class="masthead__menu-item">
                        <a href="/news/">NEWS</a>
                    </li>
                
            
                
                
                    <li class="masthead__menu-item">
                        <a href="/papers/">PAPERS</a>
                    </li>
                
            
                
                
                    <li class="masthead__menu-item">
                        <a href="/group/">GROUP</a>
                    </li>
                
            
                
                
                    <li class="masthead__menu-item">
                        <a href="/meet-the-team/">ABOUT MIN XU</a>
                    </li>
                
            
                
                
                    <li class="masthead__menu-item">
                        <a href="/position/">POSITION</a>
                    </li>
                
            
                
                
                    <li class="masthead__menu-item">
                        <a href="/software/">SOFTWARE</a>
                    </li>
                
            
        </ul>
        <!--
        
        <button class="search__toggle" type="button">
          <svg class="icon" width="16" height="16" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15.99 16">
            <path d="M15.5,13.12L13.19,10.8a1.69,1.69,0,0,0-1.28-.55l-0.06-.06A6.5,6.5,0,0,0,5.77,0,6.5,6.5,0,0,0,2.46,11.59a6.47,6.47,0,0,0,7.74.26l0.05,0.05a1.65,1.65,0,0,0,.5,1.24l2.38,2.38A1.68,1.68,0,0,0,15.5,13.12ZM6.4,2A4.41,4.41,0,1,1,2,6.4,4.43,4.43,0,0,1,6.4,2Z" transform="translate(-.01)"></path>
          </svg>
        </button>
        
        -->
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle Menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      

<div id="main" role="main">
  <article class="splash" itemscope itemtype="https://schema.org/CreativeWork">
    <meta itemprop="headline" content="Papers &amp; Preprints">
    
    
    

    <section class="page__content" itemprop="text">
      <p class="text"><strong>See <a href="https://scholar.google.com/citations?hl=en&amp;user=Y3Cqt0cAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" target="_blank">here</a> for the full list. See <a href="https://raw.githubusercontent.com/xulabs/xulabs.github.io/master/doc/reference_xulab.bib"> here </a> for corresponding citations in latex bib format.</strong></p>

<p class="text"><strong>Cryo-ET analysis</strong></p>

<ol>
        <li>
            Zeng X, Kahng A, Xue L, Mahamid J, Chang Y, Xu M. DISCA: high-throughput cryo-ET structural pattern mining by deep unsupervised clustering. bioRxiv. <a href="https://doi.org/10.1101/2021.05.16.444381" target="_blank">doi:10.1101/2021.05.16.444381</a>
        </li>
		<li>
			Gao S, Han R, Zeng X, Xu M, Zhang F. Macromolecules Structural Classification with a 3D Dilated Dense Network in Cryo-electron Tomography. IEEE/ACM Transactions on Computational Biology and Bioinformatics. <a href="https://doi.org/10.1109/TCBB.2021.3065986"> doi:10.1109/TCBB.2021.3065986 </a>
		</li>
		<li>
			Du X, Wang H, Zhu Z, Zeng X, Chang Y, Zhang J, Xing E, Xu M. <b> Active learning to classify macromolecular structures in situ for less supervision in cryo-electron tomography.</b>  Bioinformatics. <a href="https://doi.org/10.1093/bioinformatics/btab123"> doi:10.1093/bioinformatics/btab123 </a> <a href="https://arxiv.org/abs/2102.12040"> arXiv:2102.12040 </a>
		</li>
		<li>
			Zhou B, Yu H, Zeng X, Yang X, Zhang J, Xu M. One-shot Learning with Attention-guided Segmentation in Cryo-Electron Tomography. Frontiers in Molecular Biosciences. <a href="https://doi.org/10.3389/fmolb.2020.613347"> doi:10.3389/fmolb.2020.613347 </a>
		</li>
            <li>
                Liu S, Ma Y, Ban X, Zeng X, Nallapareddy V, Chaudhari A, Xu M. Efficient Cryo-Electron Tomogram Simulation of Macromolecular Crowding with Application to SARS-CoV-2. BIBM 2020
            </li>
            <li>
                Li R, Yu L, Zhou B, Zeng X, Wang Z, Yang X, Zhang J, Gao X, Jang R, Xu M. <b> Few-shot learning for classification of novel macromolecular structures in cryo-electron tomograms. PLOS Computational Biology</b>. <a href="https://doi.org/10.1371/journal.pcbi.1008227">doi:10.1371/journal.pcbi.1008227</a>
            </li>
            <li>
                Yu L, Li R, Zeng X, Wang H, Jin J, Yang G, Jiang R, Xu M. Few Shot Domain Adaptation Macromolecule Classification]{Few Shot Domain Adaptation for in situ Macromolecule Structural Classification in Cryo-electron Tomograms. Bioinformatics (2020). <a href="https://doi.org/10.1093/bioinformatics/btaa671" target="_blank"> doi:10.1093/bioinformatics/btaa671 </a>. <a href="https://arxiv.org/abs/2007.15422">arXiv:2007.15422</a>
            </li>
            <li>
                Liu S, Ban X, Zeng X, Zhao F, Gao Y, Wu W, Zhang H, Chen F, Hall T, Gao X, Xu Min. A Unified Framework for Packing Deformable and Non-deformable Subcellular Structures in Crowded Cryo-electron Tomogram Simulation. BMC Bioinformatics (2020).
            </li>
            <li>
                Gubins I, et al. SHREC'20 Benchmark: Classification in cryo-electron tomograms. Computers &amp; Graphics. 2020. <a href="https://doi.org/10.1016/j.cag.2020.07.010">doi:10.1016/j.cag.2020.07.010</a>
            </li>
            <li>
                Chen F, Jiang Y, Zeng X, Zhang J, Gao X, Xu M. PUB-SalNet: A Pre-Trained Unsupervised Self-Aware Backpropagation Network for Biomedical Salient Segmentation. Algorithms. 2020. <a href="https://doi.org/10.3390/a13050126" target="_blank"> doi:10.3390/a13050126 </a>
            </li>
            <li>
                Gao S, Han R, Zeng X, Xu M, Zhang F. Dilated-denseNet for macromolecule classification in cryo-electron tomography. International Symposium on Bioinformatics Research and Applications (ISBRA 2020). <a href="https://doi.org/10.1007/978-3-030-57821-3_8">doi:10.1007/978-3-030-57821-3_8</a>
            </li>
            <li>
                Zeng X, Xu M. <b> Gum-Net: Unsupervised geometric matching for fast and accurate 3D subtomogram image alignment and averaging</b>. In Proceedings of the IEEE conference on computer vision and pattern recognition (CVPR 2020). <a href="http://openaccess.thecvf.com/content_CVPR_2020/html/Zeng_Gum-Net_Unsupervised_Geometric_Matching_for_Fast_and_Accurate_3D_Subtomogram_CVPR_2020_paper.html" target="_blank">Paper</a>
            </li>
            <li>
                Shi J, Zeng X, Jiang R, Jiang T, Xu M. A simulated annealing approach for resolution guided homogeneous cryo electron microscopy image selection. Quantitative Biology. doi:10.1007/s40484-019-0191-8. <a href="https://link.springer.com/article/10.1007/s40484-019-0191-8" target="_blank">Paper</a>.
            </li>
            <li>
                Zeng X, Xu M. AITom: Open-source AI platform for cryo-electron tomography data analysis. (2019) <a href="https://arxiv.org/abs/1911.03044" target="_blank">arXiv:1911.03044</a>
            </li>
            <li>
                Lu Y, Zeng X, Tian X, Shi X, Wang H, Zheng X, Liu X, Zhao X, Gao X, Xu M. Spark-based parallel calculation of 3D Fourier shell correlation for macromolecule structure local resolution estimation. BMC Bioinformatics. <a href="https://doi.org/10.1186/s12859-020-03680-6"> doi:10.1186/s12859-020-03680-6 </a>
            </li>
            <li>
                Wu X, Zeng X, Zhu Z, Gao X, Xu M. (2019) Template-Based and Template-Free Approaches in Cellular Cryo-Electron Tomography Structural Pattern Mining. In: Computational Biology, (Husi H ed), Codon Publications,  <a href="https://www.google.com/url?q=https%3A%2F%2Fdx.doi.org%2F10.15586%2Fcomputationalbiology.2019.ch11&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNEXIuKWJRdyKJeULzHPDZdVJ5qQ1w" target="_blank">doi:10.15586/computationalbiology.2019.ch11</a>
            </li>
            <li>
                Che C, Xian Z, Zeng X, Gao X, Xu M. Domain Randomization for Macromolecule Structure Classification and Segmentation in Electron Cyro-tomograms. IEEE International Conference on Bioinformatics and Biomedicine (BIBM 2019)
            </li>
            <li>
                Wu X, Mao Y, Wang H, Zeng X, Gao X, Xing E, Xu M. Regularized Adversarial Training (RAT) for Robust Cellular Electron Cryo Tomograms Classification. IEEE International Conference on Bioinformatics and Biomedicine (BIBM 2019)
            </li>
            <li>
                Du X, Zeng X, Zhou B, Singh A, Xu M. <b> Open-set Recognition of Unseen Macromolecules in Cellular Electron Cryo-Tomograms by Soft Large Margin Centralized Cosine Loss</b>. British Machine Vision Conference (BMVC 2019 spotlight with acceptance rate &#60; 5%). <a href="https://www.google.com/url?q=https%3A%2F%2Fbmvc2019.org%2Fwp-content%2Fuploads%2Fpapers%2F0347-paper.pdf&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHsPGgSr-V9nd4XQgRo6XrbD-Gw3g" target="_blank">pdf</a>
            </li>
            <li>
                Liu S, Du X, Xi R, Xu F, Zeng X, Zhou B, Xu M. Semi-supervised Macromolecule Structural Classification in Cellular Electron Cryo-Tomograms using 3D Autoencoding Classifier. British Machine Vision Conference (BMVC 2019). <a href="https://www.google.com/url?q=https%3A%2F%2Fbmvc2019.org%2Fwp-content%2Fuploads%2Fpapers%2F0337-paper.pdf&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHC-ua4vqU6dKkVAI37462EuBwnhw" target="_blank">pdf</a>
            </li>
            <li>
                Lu Y, Zeng X, Zhao X, Li S, Li H, Gao X, Xu M. Fine-grained Alignment of Cryo-electron Subtomograms Based on MPI Parallel Optimization. BMC Bioinformatics (2019) 20:443. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1186%2Fs12859-019-3003-2&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHPqeNDeZFQ8DbqgD3r4fC2B6M_Eg" target="_blank">doi:10.1186/s12859-019-3003-2</a>
            </li>
            <li>
                Lin R, Zeng X, Kitani K, Xu M. Adversarial domain adaptation for cross data source macromolecule in situ structural classification in cellular electron cryo-tomograms. ISMB 2019. Bioinformatics. 2019 Jul 5; 35(14); i260–i268. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1093%2Fbioinformatics%2Fbtz364&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHxhJm3rCbHUvKF_b8lFMfKlOPxRQ" target="_blank">doi:10.1093/bioinformatics/btz364</a>
            </li>
            <li>
                Han R, Bao Z, Zeng X, Niu T, Zhang F, Xu M, Gao X. A joint method for marker-free alignment of tilt series in electron tomography. ISMB 2019. Bioinformatics. 2019. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1093%2Fbioinformatics%2Fbtz323&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNEkKBLr2mRVz5nIQWjZOwrS8qQlFg" target="_blank">doi:10.1093/bioinformatics/btz323</a>
            </li>

  <li>
                Luo Z, Zeng X, Bao Z, Xu M. Deep Learning-Based Strategy For Macromolecules Classification with Imbalanced Data from Cellular Electron Cryotomography. International Joint Conference on Neural Networks (IJCNN 2019). <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1908.09993&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNEGGz5bPVwV_mULYoK29LpmHoZcyA" target="_blank">arXiv:1908.09993</a>
            </li>
            <li>
                Li R, Zeng X, Siegmund S, Lin R, Zhou B, Liu C, Wang K, Jiang R, Freyberg Z, Lv H, Xu M. Automatic Localization and Identification of Mitochondria in Cellular Electron Cryo-Tomography using Faster-RCNN. BMC Bioinformatics. 201920 (Suppl 3) :132 <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1186%2Fs12859-019-2650-7&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHqvXvIEgkmSGKKbJRZ7YOB7ee_Gw" target="_blank">doi:10.1186/s12859-019-2650-7</a>
            </li>
            <li>
                Gubins et al. Classification in Cryo-Electron Tomograms. Eurographics Workshop on 3D Object Retrieval. doi: 10.2312/3dor.20191061
            </li>
            <li>
                Zhao G, Zhou B, Wang K, Jiang R, Xu M. <b> Respond-CAM: Analyzing Deep Models for 3D Imaging Data by Visualizations</b>. Medical Image Computing &amp; Computer Assisted Intervention (MICCAI) 2018.   <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1806.00102&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNGgP4R3YTn18pWjf3tItO1EJcBocg" target="_blank">arXiv:1806.00102</a>
            </li>
            <li>
                Liu C, Zeng X, Guo Q, Wang K, Xu M. Multi-task Learning for Macromolecule Classification, Segmentation and Coarse Structural Recovery in Cryo-Tomography. British Machine Vision Conference (BMVC) 2018. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1805.06332&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNEJAdsHh1ZV1DtigT9hzbjzi-DKfw" target="_blank">arXiv:1805.06332</a>
            </li>
            <li>
                Wang K, Zeng X, Liang X, Huo Z, Xing E, Xu M. Image-derived generative modeling of pseudo-macromolecular structures - towards statistical assessment of electron cryotomography template matching. British Machine Vision Conference (BMVC) 2018. <a href="http://www.bmva.org/bmvc/2018/contents/papers/0532.pdf"> pdf </a>. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1805.04634&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNGKj3NyL-PIBryMQi3zOw1_EJZ6fw" target="_blank">arXiv:1805.04634</a>
            </li>
            <li>
                Liu C, Zeng X, Lin R, Liang X, Freyberg Z, Xing E, Xu M. Deep learning based supervised semantic segmentation of Electron Cryo-Subtomograms. IEEE International Conference on Image Processing (ICIP) 2018. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1802.04087&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNE5BalTN63Z8cWmMj0qAIiUxCb7Qw" target="_blank">arXiv:1802.04087</a>
            </li>
            <li>
                Zhao Y, Zeng X, Guo Q, Xu M. <b> An integration of fast alignment and maximum-likelihood methods for electron subtomogram averaging and classification</b>. ISMB 2018. Bioinformatics. 2018 Jul 1; 34(13): i227–i236. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1093%2Fbioinformatics%2Fbty267&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNEzaiJD5hLMLjD4MIKW2O9gJ-Icvg" target="_blank">doi:10.1093/bioinformatics/bty267</a>. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1804.01203&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFUZrHiXv_yh0KWf1DO4qAy1IHOWQ" target="_blank">arXiv:1804.01203</a>
            </li>
            <li>
                Guo J, Zhou B, Zeng X, Freyberg Z, Xu M. Model compression for faster structural separation of macromolecules captured by Cellular Electron Cryo-Tomography.  <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1801.10597&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFQRg0GTEN8s2EaJSww7Dc-lCZHBg" target="_blank">arXiv:1801.10597</a>. International Conference on Image Analysis and Recognition (ICIAR) 2018
            </li>
            <li>
                Zhou B, Guo Q, Zeng X, Gao X, Xu M. Feature Decomposition Based Saliency Detection in Electron Cryo-Tomograms. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1801.10562&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHaPv2_pv7s6zdRSGiM8levJDld8Q">arXiv:1801.10562</a>. IEEE International Conference on Bioinformatics &amp; Biomedicine, Workshop on Machine Learning in High Resolution Microscopy (BIBM-MLHRM 2018)
            </li>
            <li>
                Che C, Lin R, Zeng X, Elmaaroufi K, Galeotti J, Xu M. Improved deep learning based macromolecules structure classification from electron cryo tomograms. Machine vision and applications 29.8 (2018): 1227-1236. doi:10.1007/s00138-018-0949-4.  <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1707.04885&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFVQ0NVTvzjbm5EVYnmj94Wmh1ZvQ" target="_blank">arXiv:1707.04885</a>
            </li>
            <li>
                Zeng X, Leung M, Zeev-Ben-Mordehai T, Xu M. <b> A convolutional autoencoder approach for mining features in cellular electron cryo-tomograms and weakly supervised coarse segmentation </b>. Journal of Structural Biology.  2018 May;202(2):150-160. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1016%2Fj.jsb.2017.12.015&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHd6jJBB-Obp_ebGPvYCwwXv9UrkA" target="_blank">doi:10.1016/j.jsb.2017.12.015</a>  <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1706.04970&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFez1S5-gvM5whMCXe1yU5-vXVa2g" target="_blank">arXiv:1706.04970</a> [<a href="https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fxulabs%2Fprojects%2Ftree%2Fmaster%2Fautoencoder&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNG3kvV1s4UAW8nIBaqZN67rbqN-uA">code</a>]
            </li>
            <li>
                Xu M, Chai X, Muthakana H, Liang X, Yang G, Zeev-Ben-Mordehai T, Xing E. Deep learning based subdivision approach for large scale macromolecules structure recovery from electron cryo tomograms. 2017. <a href="https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1701.08404&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFIv5ZZ4mZCfToeLMsuslJpZ1s2Aw" target="_blank">arXiv:1701.08404</a>.  ISMB 2017 (acceptance rate 16%), Bioinformatics <a href="http://www.google.com/url?q=http%3A%2F%2Fdx.doi.org%2F10.1093%2Fbioinformatics%2Fbtx230&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNGmko1GllIVm8-m-l30YjSPRVFS9w" target="_blank">doi:10.1093/bioinformatics/btx230</a> [<a href="https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2Fxulabs%2Fprojects%2Fblob%2Fmaster%2Fparticle_subdivision%2Fsubdivide.py&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNGBfTfETvntmRlmsGiFvDXb8q_EgA" target="_blank">code</a>]
            </li>
            <li>
                Xu M, Singla J, Tocheva E, Chang Y, Stevens R, Jensen G, Alber F. <b> De novo structural pattern mining in cellular electron cryo-tomograms</b>. Structure. 2019 Apr 2;27(4):679-691.e14.  <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1016%2Fj.str.2019.01.005&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNG4CwvStj7Z7IutfOKq_h8TcKdxZw" target="_blank">doi:10.1016/j.str.2019.01.005</a>.  (Appeared on Structure volume cover and highlighted in Nature Methods 16, page 285 (2019), <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1038%2Fs41592-019-0382-2&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHMCQyejm_T4RGn8t5EKPadA4XFzA" target="_blank">doi:10.1038/s41592-019-0382-2</a>)
            </li>
            <li>
                Frazier Z, Xu M, Alber F. TomoMiner and TomoMiner Cloud: A software platform for large-scale subtomogram structural analysis. Structure 2017. (doi: 10.1016/j.str.2017.04.016, co-corresponding author).
            </li>
            <li>
                Pei L, Xu M, Frazier Z, Alber F. Simulating Cryo-Electron Tomograms of Crowded Mixtures of Macromolecular Complexes and Assessment of Particle Picking. BMC Bioinformatics. 2016; 17: 405.
            </li>
            <li>
                Xu M, Alber F. Automated target segmentation and real space fast alignment methods for high-throughput classification and averaging of crowded cryo-electron subtomograms. ISMB/ECCB 2013, Bioinformatics. 2013 Jul 1;29(13):i274-82.
            </li>
            <li>
                Thalassinos K, Pandurangan AP, Xu M, Alber F, Topf M. Conformational States of macromolecular assemblies explored by integrative structure calculation. Structure. 2013 Sep 3;21(9):1500-8.
            </li>
            <li>
                Xu M, Beck M, Alber F. <b> High-throughput subtomogram alignment and classification by Fourier space constrained fast volumetric matching</b>. Journal of Structural Biology. 2012 May;178(2):152-64. Epub 2012 Mar 7.
            </li>
            <li>
                Xu M, Alber F. High precision alignment of cryo-electron subtomograms through gradient-based parallel optimization. BMC Systems Biology. 2012, 6(Suppl 1):S18
            </li>
            <li>
                Xu M, Beck M, Alber F. Template-free detection of macromolecular complexes in cryo-electron tomograms. Bioinformatics (ISMB 2011). 2011 Jul 1;27(13):i69-i76.
            </li>
            <li>
                Beck M, Topf M, Frazier Z, Tjong H, Xu M, Zhang S, Alber F. Exploring the Spatial and Temporal Organization of a Cell’s Proteome. Journal of Structural Biology. 2011 Mar; 173(3):483-496.
            </li>
            <li>
                Zhang S, Vasishtan D, Xu M, Topf M, Alber F. A fast mathematical programming procedure for simultaneous fitting of assembly components into cryo-EM density maps. Bioinformatics (ISMB 2010). 2010 Jun 15;26(12):i261-8.
            </li>
            <li>
                Xu M, Zhang S, Alber F. 3D rotation invariant features for characterization of molecular density map images. IEEE International Conference on Bioinformatics &amp; Biomedicine (BIBM 2009)
            </li>
        </ol>

<p class="text"><strong>Computer vision &amp; Machine learning</strong></p>
<ol>
      <li>
          Yang Y, Ma Y, Zhang J, Gao X, Xu M. AttPNet: Attention-Based Deep Neural Network for 3D Point Set Analysis. Sensors. <a href="https://doi.org/10.3390/s20195455" target="_blank"> doi:10.3390/s20195455 </a>
      </li>
      <li>
          Pang J, Yi K, Yin W, Xu M. Experimental Analysis of Legendre Decomposition in Machine Learning. <a href="https://arxiv.org/abs/2008.05095"> arXiv:2008.05095 </a>
      </li>
  </ol>

<p class="text"><strong>Biomedical image analysis</strong></p>

<ol>
            <li>
                Dong N, Kampffmeyer M, Liang X, Xu M, Voiculescu I, Xing E. Towards Robust Medical Image Segmentation on Small-Scale Data with Incomplete Labels. <a href="https://arxiv.org/abs/2011.14164">arXiv:2011.14164</a>
            </li>
            <li>
                Dong N, Xu M, Liang X, Jiang Y, Dai W, Xing E. Neural Architecture Search for Adversarial Medical Image Segmentation. Medical Image Computing &amp; Computer Assisted Intervention (MICCAI 2019).
            </li>
            <li>
                Xiao Q, Zou J, Yang M, Gaudio A, Kitani K, Smailagic A, Costa P, Xu M. Improving Lesion Segmentation for Diabetic Retinopathy Using Adversarial Learning. International Conference on Image Analysis and Recognition (ICIAR 2019)
            </li>
            <li>
                Wang W, Taft D, Chen Y, Zhang J, Wallace C, Xu M, Watkins S, Xing J. Learn to segment single cells with deep distance estimator and deep cell detector. Computers in Biology and Medicine. <a href="https://www.google.com/url?q=https%3A%2F%2Fdoi.org%2F10.1016%2Fj.compbiomed.2019.04.006&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNF0Y1Lx7OAEalJ5fy5e-PGkbppwuQ" target="_blank">doi:10.1016/j.compbiomed.2019.04.006</a>
            </li>
            <li>
                Wang Z, Dong N, DRosario S, Xu M, Xie P, Xing E. Ellipse detection of optic disc-and-cup boundary in fundas image with unsupervised domain adaptation. IEEE International Symposium on Biomedical Imaging (ISBI 2019)
            </li>
        </ol>

<p class="text"><strong>Genomics</strong></p>
<ol>
  <li>
    Chen Z, Zhang J, Liu J, Zhang Z, Zhu J, Xu M, Gerstein M. SCAN-ATAC-Sim: a scalable and efficient method for simulating single-cell ATAC-seq data from bulk-tissue experiments. Bioinformatics.
  </li>
          <li>
              Zheng Y, Wang H, Zhang Y, Gao X, Xing E, Xu M. Poly(A)-DG: a deep-learning-based domain generalization method to identify cross-species Ploy(A) signal without prior knowledge from target species. PLOS Computational Biology. <a href="https://doi.org/10.1371/journal.pcbi.1008297">doi:10.1371/journal.pcbi.1008297</a>
          </li>
          <li>
              Xu M, Li W, James GM, Mehan MR, Zhou XJ. <b> Automated multidimensional phenotype profiling using large public microarray repositories</b>. Proc Natl Acad Sci U S A. (PNAS). 2009; 106(30), 12323 - 12328. (Highlighted in Nature Methods 6, 632; Selected and re-published in 2010 International Medical Informatics Association Yearbook of Medical Informatics)
          </li>
          <li>
              Xu M, Kao MJ, Nunez-Iglesias J, Nevins JR, West M, Zhou XJ. An integrative approach to characterize disease-specific pathways and their coordination: a case study in cancer. BMC Genomics. 2008; 9 Suppl 1:S12.
          </li>
          <li>
              Xu M, Zhu M, Zhang L. A stable iterative method for refining discriminative gene clusters. BMC Genomics. 2008 Sep 16;9 Suppl 2:S18
          </li>
      </ol>

<p class="text"><strong>Computational biology</strong></p>
<ol>
    <li>
        Wang H, Wei Y, Cao M, Xu M, Wu W, Xing E. Deep Inductive Matrix Completion for Biomedical Interaction Prediction. IEEE International Conference on Bioinformatics and Biomedicine (BIBM 2019, acceptance rage 18%)
    </li>
</ol>


    </section>
  </article>
</div>

    </div>

    
      <div class="search-content">
        <div class="search-content__inner-wrap"><form class="search-content__form" onkeydown="return event.key != 'Enter';">
    <label class="sr-only" for="search">
      Enter your search term...
    </label>
    <input type="search" id="search" class="search-input" tabindex="-1" placeholder="Enter your search term..." />
  </form>
  <div id="results" class="results"></div></div>

      </div>
    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js"></script>




<script src="/assets/js/lunr/lunr.min.js"></script>
<script src="/assets/js/lunr/lunr-store.js"></script>
<script src="/assets/js/lunr/lunr-en.js"></script>







  </body>
</html>
